.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+*~.rb
```

Comment:
Well, it was _inspired_ by an editor-specific temp file from gedit on my linux box. I was just using that to easily hide all my unfinished dessert files (board & dictionary class definitions & specs) from git so I could commit the rest all at once. But I will definitely look into updating my global gitignore, since I use this `~ == hidden` syntax a lot. Thank you!